### ISSUE TITLE
Tangent preferences panel bug when assigning actions with single quotes in the action title

### ISSUE THREAD

**Issue by latenitefilms** [fcpxhacks/fcpxhacks#2449]
Reported by Vigneswaran Rajkumar.

```lua
2020-09-08 15:37:56: 15:37:56   prefsMgr:     Javascript Error: {
  code = 4,
  domain = "WKErrorDomain",
  localizedDescription = "A JavaScript exception occurred"
}
Caused by script: setTangentAction(2.0, '' + Fn⌃⌥⌘')
```

**Comment by latenitefilms** [comment]
Also, the "Clear" buttons actually incorrectly clear EVERYTHING.

**Comment by latenitefilms** [comment]
Fixed in #2442
